src/System.Security.Cryptography.Cng/tests/System.Security.Cryptography.Cng.Tests.csproj

```diff
@@ -30,7 +30,7 @@
     <Compile Include="OpenTests.cs" />
     <Compile Include="ImportExportTests.cs" />
     <Compile Include="PropertyTests.cs" />
-    <Compile Include="RSACngTests.cs" />
```

Comment:
No.  This is the casing of the file on disc.  Without these changes, MSBuild complains about being unable to find files.